### PR TITLE
Portfolio website responsive design

### PR DESCRIPTION
**Pull request by KMT-Media**
In this pull request the following actions are implemented:
* Responsive design using media queries.
* Flexbox and CSS grid are used to lay out the elements inside the web page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,20 +10,11 @@
...
-            fill="#6070FF"
-          />
-        </svg>
+        <ul class="nav-menu">
+          <li><a href="">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
       </nav>
     </header>
```
Comment (bienvenuushindi) [inline index.html]: :red_circle:  Please make sure you move  `Glad to see you!` to a new line you can insert a` <br>` tag between `I'm Mariane` and `Glad to see you!` to insert a single line break. Your final code will look like this `I'm Mariane <br> Glad to see you!`

Your Design
headertitle

Figma Design
titleheadline

Comment (bienvenuushindi) [reply index.html]: :red_circle: Kindly also adapt the border radius at the bottom left  section to match the Figma design

Your Design

borderradius


Figma Design

borderfigma

File: index.html [hunk 2]
```diff
@@ -242,207 +229,153 @@ <h2 class="primary-text">Multi-Post Stories</h2>
...
-              <div class="language-list-item">
-                <img
-                  class="language-list-item-image"
-                  src="./assets/images/js.png"
-                  alt="Javascript image"
-                />
-                <p class="language-list-item-text">Javascript</p>
-              </div>
+      <div class="about-left">
+        <h2 class="primary-text">About Myself</h2>
```
Comment (bienvenuushindi) [inline index.html]: :red_circle:  Please make sure you move  `Myself` to a new line you can insert a` <br>` tag between `About` and `Myself` to insert a single line break. Your final code will look like this `About <br> Myself`

Your Design

titleStudent

Figma Design

titlefigma

File: index.html [hunk 3]
```diff
@@ -242,207 +229,153 @@ <h2 class="primary-text">Multi-Post Stories</h2>
...
+            </a>
+          </li>
+        </ul>
+        <button type="button" class="btn">Get my resume</button>
+      </div>
+      <div class="about-right">
+        <ul class="drop-options">
+          <li class="drop-options-list">
+            <ul class="hidden-options-title">
+              <li>Languages</li>
```
Comment (bienvenuushindi) [inline index.html:285]: :red_circle: Please make sure you pick the right font color from the Figma template. As for now, Your font color does not reflect the font color in the Figma Design

Your Design
fontcolorstudent


Figma Design

fontcolor

File: index.html [hunk 4]
```diff
@@ -242,207 +229,153 @@ <h2 class="primary-text">Multi-Post Stories</h2>
...
+                  <img
+                    class="language-list-item-image"
+                    src="./assets/images/css.png"
+                    alt="Javascript image"
+                  />
+                  <p class="language-list-item-text">CSS</p>
+                </div>
+              </a>
+            </ul>
+          </li>
```
Comment (bienvenuushindi) [inline index.html:335]: :red_circle:   Kindly reduce your cards' width and leave some room at the right of the languages list to match the Figma design

Your Design
tech

Figma Design
tecnologie list

File: index.html [hunk 5]
```diff
@@ -242,207 +229,153 @@ <h2 class="primary-text">Multi-Post Stories</h2>
...
+          <li class="drop-options-list">
+            <ul class="hidden-options-title">
```
Comment (bienvenuushindi) [inline index.html:337]: - [ ] :red_circle:  As you can see from the Figma design, there is a top margin between the technology categories( Frameworks) and the languages list.  Kindly fix this to match the Figma design

Your Design
nospace

Figma Design
spacing

File: style.css [hunk 6]
```diff
@@ -331,3 +347,115 @@ input:focus {
   color: #fff;
   transition: all 0.8s;
 }
+
+@media (min-width: 768px) {
```
Comment (bienvenuushindi) [inline style.css:368]: - [ ] :red_circle:  The mobile version of your portfolio seems to be affected by the modifications you have made. Kindly try to fix these issues. 
  -  The content does not fit and you have to use the horizontal scroll bar to see everything
  -  Please make sure you set a fixed width on each image in your cards.
  -  Please make sure on the mobile screen, only the hamburger menu appears and not the menu items


Your Mobile version

affectected

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
### Hi @KMT-Media ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linter checks are ok. ✔
- 

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- { ] N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by aimalamiri — APPROVED** [review]
Hi @KMT-Media, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

ApprovedDavieGIF


## Highlights

- The desktop version is designed very well and looks fantastic ✔️ ⭐
- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍
 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**